### DISCUSSION TITLE
Can’t edit newly created issue

### DISCUSSION THREAD

**Discussion by ghost** [community/community#64434] [Mobile]
When creating an issue in the latest version, I am unable to edit the body text in the app. I am forced to switch to the web version.

> jslawler does not have permission to update the issue I_kwDOCZ_oDc5uwpoy.

See below for error.
![image](https://github.com/community/community/assets/95468948/05bf0658-ea0b-4706-a137-4d824c5a6bc8)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.126.0 (107057637)
Device: iPhone 13 Pro (iOS 17.0)
</details>

**Comment by rizul11** [comment]
[https://github.com/orgs/community/discussions/3970](url) 

Check this out

**Reply by ghost** [reply]
Seems like a mostly dead issue tracker. I've left a message anyway.
It was originally created for Android, so I'll leave this open.
